relay-event-normalization/src/normalize/processor.rs

```diff
@@ -224,9 +224,6 @@ impl<'a> Processor for NormalizeProcessor<'a> {
             return Ok(());
         }
 
-        // Check for required and non-empty values
-        schema::SchemaProcessor.process_event(event, meta, ProcessingState::root())?;
-
```

Comment:
Does the order when the schema processor was running and how the check runs now matters? 
Does it execute in the same order? Different?

Reply:
The order is different: `SchemaProcessor` actions now run as part of processing child values. This doesn't matter, as `SchemaProcessor` adds remarks and/or meta errors on missing values. Decisions on rejecting events stay the same, and normalization steps won't be performed as the value is missing (same behavior).

I updated the PR description to reflect this.